### ISSUE TITLE
Missing audio when concurrently downloading and using webgui

### ISSUE THREAD

**Issue by Inexpress** [nilfoer/gwaripper#9]
I had an issue recently where I attempted to download an audio while the webgui was running in another window. The audio file was downloaded, and gwaripper recognises it's there, refusing to grab it again, but it doesn't appear in the webgui. Not sure if this is an issue unique in some way to this audio or a more general bug. (https://www.reddit.com/r/gonewildaudio/comments/xzzgfh/tm4m_m4m_pupcup_cafe_would_you_like_the_special/)

**Comment by nilfoer** [comment]
I didn't test all interactions to be safe, so I wouldn't recommend running the webgui and downloading new files etc. at the same time.

Testing your link worked for me using the following steps:

1. started the webgui
2. downloaded the file
3. refreshed the page and the audio showed up

It would help if you could find or describe a way to reproduce the problem. As a note though: the webgui server will not notify the client/your open page about new audios, the page has to be refreshed in order to fetch the new information.

**Comment by Inexpress** [comment]
Thanks for your response. As things currently stand, I've downloaded other audios since (without the webgui running) and all have been downloaded normally. I also ran a test trying to download an arbitrary audio with the webgui running and it worked perfectly fine. So I haven't been able to reproduce the issue. But if there are any debugging/logging steps I can take to help shed some light on what's going on here I am happy to help.
